WooCommerce/Classes/ViewRelated/Blaze/AdDestination/BlazeAdDestinationSettingView.swift

```diff
@@ -0,0 +1,222 @@
+import SwiftUI
+
+/// View to set ad destination for a new Blaze campaign
+struct BlazeAdDestinationSettingView: View {
+    @Environment(\.dismiss) private var dismiss
+
+    @ObservedObject private var viewModel: BlazeAdDestinationSettingViewModel
+
+    typealias DestinationType = BlazeAdDestinationSettingViewModel.DestinationURLType
+
+    init(viewModel: BlazeAdDestinationSettingViewModel) {
+        self.viewModel = viewModel
+    }
+
+    var body: some View {
+        NavigationView {
+            VStack(alignment: .leading, spacing: 0) {
+                // URL destination section
+                VStack(spacing: 0) {
+                    sectionHeading(title: Localization.destinationUrlHeading)
+                    destinationItem(title: Localization.productURLLabel,
+                                    subtitle: String(format: Localization.destinationUrlSubtitle, viewModel.productURL),
+                                    type: DestinationType.product,
+                                    showBottomDivider: true)
+
+                    destinationItem(title: Localization.siteHomeLabel,
+                                    subtitle: String(format: Localization.destinationUrlSubtitle, viewModel.homeURL),
+                                    type: DestinationType.home)
+                }
+                .padding(.bottom, Layout.sectionVerticalSpacing)
+
+                // URL parameters section
+                VStack {
+                    sectionHeading(title: Localization.urlParametersHeading)
+
+                    VStack {
+                        ForEach(viewModel.parameters, id: \.self) { parameter in
+                            parameterItem(itemName: parameter.key)
+                        }
+
+                        Button(Localization.addParameterButton) {
+                            // todo
+                        }
+                        .buttonStyle(PlusButtonStyle())
```

Comment:
known issue: this uses an existing style that's also used in other places in the app. However, this makes the plus icon look slightly bigger than in the Blaze design.

Reply:
At first I tried refactoring `PlusButtonStyle` but it seems non-trivial, because it (and the underlying `IconButton`) is used in other places already.

To save time I ended up just refactoring this particular button only to use SFSymbol. Handled in the next PR: https://github.com/woocommerce/woocommerce-ios/pull/11642/commits/7b7d575dadd8597c442c6896c48ee12f57d75533